docs/Developers/Writing_Good_Commits.rst

```diff
@@ -120,6 +120,11 @@ Example:
 
 -  Should use the ``Fixes`` keyword if your commit fixes a bug, or ``Closes``
    if it adds a feature/enhancement.
+-  In some certain situations, for example bugs overcome in documents, the
```

Comment:
instead of `for example` use `e.g.` we use it consistently so I think we better stick with it